dallinger/command_line/__init__.py

```diff
@@ -650,12 +650,31 @@ def awaken(app, databaseurl):
 
 @dallinger.command()
 @click.option("--app", default=None, callback=verify_id, help="Experiment id")
-@click.option("--local", is_flag=True, flag_value=True, help="Export local data")
-@click.option("--no-scrub", is_flag=True, flag_value=True, help="Scrub PII")
+@click.option(
+    "--local",
+    is_flag=True,
+    flag_value=True,
+    help="Only export data locally, skipping the Amazon S3 copy",
+)
+@click.option(
+    "--no-scrub",
+    is_flag=True,
+    flag_value=True,
+    help="Scrub PII (Personally Identifiable Information)",
```

Comment:
I propose "Don't scrub PII (Personally Identifiable Information) - if not specified PII will be scrubbed"

Reply:
💯